x-pack/filebeat/docs/inputs/input-http-endpoint.asciidoc

```diff
@@ -171,6 +180,40 @@ Preserving original event and including headers in document
   include_headers: ["TestHeader"]
 ----
 
+Common Expression Language example:
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: http_endpoint
+  enabled: true
+  listen_address: 192.168.1.1
```

Comment:
[Suggestion]
Use the loopback IP address so people can copy/paste the config as is for testing/experimenting.
```suggestion
  listen_address: 127.0.0.1
```

Reply:
I think in this case documentational consistency and clarity trumps copy/paste convenience; the other examples all use 192.168.1.1 and that address is an address that is not special. Changing either of those things I think harms clarity.